styleguide.config.js

```diff
@@ -154,6 +154,11 @@ module.exports = {
             name: "Spacing",
             content: "packages/wonder-blocks-spacing/docs.md",
         },
+        {
+            private: true,
+            name: "Timing",
```

Comment:
IMO, I think this might make sense as an external package - but having this be in WB for now is totally fine by me!

Reply:
Yes, this private should be in the package.json. I'll make a change.

Once the package is finished, we can make it public.